Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,14 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/tag-4.9.25.tar.gz"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
```

Comment:
Can you please submit this as an issue to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!